package.json

```diff
@@ -1,42 +1,61 @@
 {
-  "name": "minecraft-launcher-core",
-  "version": "3.18.1",
-  "description": "Lightweight module that downloads and runs Minecraft using javascript / NodeJS",
-  "main": "index.js",
-  "dependencies": {
-    "adm-zip": "^0.4.13",
-    "checksum": "^0.1.1",
-    "request": "^2.88.2",
-    "uuid": "^8.3.2"
-  },
-  "devDependencies": {
-    "@types/checksum": "^0.1.35",
-    "@types/node": "^14.0.27",
-    "eslint": "^6.8.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1"
-  },
-  "scripts": {
-    "test": "eslint ."
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Pierce01/MinecraftLauncher-core.git"
-  },
-  "keywords": [
-    "minecraft",
-    "minecraft-launcher-node",
-    "minecraft-launcher",
-    "launcher",
-    "minecraft-forge"
-  ],
-  "author": "Pierce Harriz",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/Pierce01/MinecraftLauncher-core/issues"
-  },
-  "homepage": "https://github.com/Pierce01/MinecraftLauncher-core#readme"
+    "name": "minecraft-launcher-core",
+    "version": "3.19.0",
+    "description": "Lightweight module that downloads and runs Minecraft using javascript / NodeJS",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Pierce01/MinecraftLauncher-core.git"
+    },
+    "keywords": [
+        "minecraft",
+        "minecraft-launcher-node",
+        "minecraft-launcher",
+        "launcher",
+        "minecraft-forge"
+    ],
+    "author": "Pierce Harriz",
+    "funding": "https://github.com/Pierce01/MinecraftLauncher-core?sponsor=1",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/Pierce01/MinecraftLauncher-core/issues"
+    },
+    "homepage": "https://github.com/Pierce01/MinecraftLauncher-core#readme",
+    "main": "./build/cjs/index.js",
+    "module": "./build/esm/index.js",
+    "types": "./build/esm/index.d.ts",
+    "files": [
+        "./build/"
+    ],
+    "type": "module",
+    "scripts": {
+        "clean": "rm -rf ./lib/",
+        "build": "yarn clean && yarn build:esm && yarn tsc",
```

Comment:
Why yarn ? The original project was working on NPM only

Reply:
npm isn't that good, while Yarn is better. pnpm might be even better, but totally not npm

Reply:
Let Pierce decide on this part